tests/v2/validation/provisioning/k3s/README.md

```diff
@@ -8,118 +8,129 @@ Please see below for more details for your config. Please see below for more det
 
 ## Table of Contents
 1. [Prerequisites](../README.md)
-2. [Define your test](#provisioning-input)
-3. [Cloud Credential](#cloud-credentials)
-4. [Configure providers to use for Node Driver Clusters](#machine-k3s-config)
-5. [Configuring Custom Clusters](#custom-cluster)
-6. [Static test cases](#static-test-cases) 
-7. [Advanced Cluster Settings](#advanced-settings)
-8. [Back to general provisioning](../README.md)
+2. [Configuring test flags](#Flags)
+3. [Define your test](#provisioning-input)
+4. [Cloud Credential](#cloud-credentials)
+5. [Configure providers to use for Node Driver Clusters](#machine-k3s-config)
+6. [Configuring Custom Clusters](#custom-cluster)
+7. [Static test cases](#static-test-cases)
+8. [Advanced Cluster Settings](#advanced-settings)
+9. [Back to general provisioning](../README.md)
+
+## Flags
+Flags are used to determine which static table tests are run (has no effect on dynamic tests) 
+`Long` Will run the long version of the table tests (usually all of them)
+`Short` Will run the subset of table tests with the short flag.
+
+```yaml
+flags:
+  desiredflags: "Long"   #required (static tests only)
+```
 
 ## Provisioning Input
-provisioningInput is needed to the run the K3S tests, specifically kubernetesVersion and providers. nodesAndRoles is only needed for the TestProvisioningDynamicInput test, node pools are divided by "{nodepool},". psact is optional and takes values `rancher-privileged`, `rancher-restricted` or `rancher-baseline`.
+provisioningInput is needed to the run the K3S tests, specifically kubernetesVersion and providers.
 
 **nodeProviders is only needed for custom cluster tests; the framework only supports custom clusters through aws/ec2 instances.**
 ```yaml
 provisioningInput:
   machinePools:
-  - nodeRoles:
-      etcd: true
+  - nodeRoles:                                              #required(dynamic only) (at least 1)
+      etcd: true                                            #required(dynamic only) (at least 1 controlplane & etcd & worker)
```

Comment:
nit; I find the comment a little confusing but I can't think of a way to make it better 😓

Reply:
Agreed it isn't the most clear. But as you said I don't know how to make it much clearer without a full explination.